### PR TITLE
use limit switches on indexer

### PR DESCRIPTION
**Pull request by lmaxwell24**

### CODE REVIEW

File: subsystems/indexersubsystem.py [hunk 1]
```diff
@@ -131,14 +131,18 @@ def periodic(self) -> None:
...
         elif self.state == self.Mode.Holding:
-            self.indexerMotor.set(
-                ControlMode.Velocity,
-                constants.kIndexerSpeed * constants.kTalonVelocityPerRPM,
-            )
-            self.stagingMotor.set(
-                ControlMode.Velocity,
-                -constants.kStagingSpeed * constants.kTalonVelocityPerRPM,
-            )
+            if not (self.indexerSensor() == 0 and self.stagingSensor() == 0):
```
Comment (LandonBayer) [inline subsystems/indexersubsystem.py:134]: Why not if (resolves true)? Is there a reason this is specifically if NOT instead?

**Review by LandonBayer — APPROVED** [review]
It'll work but I am a bit confused

File: subsystems/indexersubsystem.py (hunk 1 above)
Comment (caloisio) [reply subsystems/indexersubsystem.py:134]: Can shorten to `if (self.indexerSensor() != 0 or self.stagingSensor() != 0)`

**Review by caloisio — APPROVED** [review]

File: subsystems/indexersubsystem.py (hunk 1 above)
Comment (LandonBayer) [reply subsystems/indexersubsystem.py:134]: This probably doesn't really make a performance difference so I'm gonna skip over it for now

File: subsystems/indexersubsystem.py [hunk 2]
```diff
@@ -131,14 +131,18 @@ def periodic(self) -> None:
...
+                self.indexerMotor.set(
+                    ControlMode.Velocity,
+                    constants.kIndexerSpeed * constants.kTalonVelocityPerRPM,
+                )
+                self.stagingMotor.set(
+                    ControlMode.Velocity,
+                    -constants.kStagingSpeed * constants.kTalonVelocityPerRPM,
+                )
+            else:
+                self.indexerMotor.set(ControlMode.Velocity, 0)
```
Comment (LandonBayer) [inline subsystems/indexersubsystem.py:144]: I see the commit changing this out of neutralOutput: why?

**Review by LandonBayer — APPROVED** [review]

**Pull request merged**